used_jsons/stargazers.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
let's move top100 repos to readme
and this file should be created in directory `results`